src/EPR.Accreditation.Facade/Services/AccreditationMaterialService.cs

```diff
@@ -142,5 +141,41 @@ public class AccreditationMaterialService : IAccreditationMaterialService
                 materialId,
                 material);
         }
+
+        public async Task<bool?> GetHasNpwdAccreditationNumber(
+            Guid accreditationExternalId,
```

Comment:
Remove external from param names. Then rename accreditationId to id

Reply:
Changed